### PR TITLE
Home

### PR DESCRIPTION
**Pull request by Digambar1411**
feat: added home page 
ignore the commented code as of now, will remove at the end

files to check

frontend/components/footer.js
frontend/components/nav.js
frontend/pages/home.js
frontend/pages/home.css

image

app.js
index.js

### CODE REVIEW

File: src/frontend/pages/home/home.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { Link } from "react-router-dom";
+import { Footer } from "../../components/footer/footer";
+import { Navbar } from "../../components/navbar/nav";
```
Comment (junaidshaikh-js) [inline src/frontend/pages/home/home.js]: barrel imports can be used here for the component folder

**Review by junaidshaikh-js — APPROVED** [review]
Good work. 

Suggested one change, everything else looks fine and can be merged.

File: src/frontend/pages/home/home.js (hunk 1 above)
Comment (Digambar1411) [reply src/frontend/pages/home/home.js]: Yeah thanks will do it

**Comment by som749898** [comment]
looks good to merge

**Review by arpit01923 — COMMENTED** [review]
- clean UI
- good folder structure
- one small suggestion, try barrel exports as well as absolute paths

**Pull request merged**